### PR TITLE
new external starlight added

### PR DESCRIPTION
**Pull request by smuzaffar**
same as #2049 with updated tag/hash of starlight

### CODE REVIEW

**Comment by smuzaffar** [comment]
please test

File: starlight.spec [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ -DENABLE_CLHEP=ON
+
+make %{makeprocesses} VERBOSE=1
+
+%install
+cd ../build
+make %{makeprocesses} install VERBOSE=1
+
+rm -rf %{i}/lib/archive
+rm -rf %{i}/lib/libStarlib.a
```
Comment (davidlt) [inline starlight.spec]: Remove this line, there is no such file.

```
a/slc6_amd64_gcc493/external/starlight/r193-cms/lib
a/slc6_amd64_gcc493/external/starlight/r193-cms/lib/libStarlib.so
a/slc6_amd64_gcc493/external/starlight/r193-cms/lib/archive
a/slc6_amd64_gcc493/external/starlight/r193-cms/lib/archive/libStarlib.a
```

Other than that, LGTM.

**Comment by smuzaffar** [comment]
merge
looks good

**Comment by smuzaffar** [comment]
@KiSooLee , starlight does not build with gcc 5.3.0. this is breaking gcc530 IBs. 

```
In file included from /build/build/muz/sl1/BUILD/slc6_amd64_gcc530/external/starlight/r193/starlight-r193/src/randomgenerator.cpp:39:0:
/build/build/muz/sl1/slc6_amd64_gcc530/external/clhep/2.2.0.4-ikhhed/include/CLHEP/Random/Randomize.h:70:18: error: 'CLHEP::HepRandomGenActive' defined but not used [-Werror=unused-variable]
 static const int HepRandomGenActive = HepRandom::createInstance();
                  ^
cc1plus: all warnings being treated as errors
```

**Comment by davidlt** [comment]
I will take care of this, there is not much be done done (`CXXFLAGS = -Wno-error=unused-variable`).

**Comment by davidlt** [comment]
Solved.

**Comment by smuzaffar** [comment]
Thanks @davidlt, but I mistakenly merged it. This still needs to be discussed in ORP. So I am removing it from cmssw-tool-conf

**Comment by davidlt** [comment]
I wouldn't remove it. It does not affect CMSSW in any way. Wait for the decision in ORP.

**Comment by smuzaffar** [comment]
too late, already done :-)

**Pull request merged**